src/logistration/RegistrationPage.jsx

```diff
@@ -353,34 +349,33 @@ class RegistrationPage extends React.Component {
     const fields = this.props.formData.fields.map((field) => {
       let options = null;
       if (REGISTRATION_EXTRA_FIELDS.includes(field.name)) {
-        if (!field.required) {
+        if (!field.required && field.name !== 'honor_code' && field.name !== 'country') {
           REGISTRATION_OPTIONAL_MAP[field.name] = true;
           const props = {
             id: field.name,
             name: field.name,
             type: field.type,
             onChange: e => this.handleOnChange(e),
           };
-          if (field.name !== 'honor_code' && field.name !== 'country') {
-            if (field.type === 'select') {
-              options = field.options.map((item) => ({
-                value: item.value,
-                label: item.name,
-              }));
-              props.options = options;
-            }
-            return (
-              <ValidationFormGroup
-                for={field.name}
-              >
-                <label htmlFor={field.name} className="h6 pt-3">{field.label} (optional)</label>
-                <Input {...props} />
-              </ValidationFormGroup>
-            );
+          if (field.type === 'select') {
+            options = field.options.map((item) => ({
+              value: item.value,
+              label: item.name,
+            }));
+            props.options = options;
           }
+          return (
+            <ValidationFormGroup
+              for={field.name}
+              key={field.name}
+            >
+              <label htmlFor={field.name} className="h6 pt-3">{field.label} (optional)</label>
```

Comment:
do we need to mark (optional) as translation string?

Reply:
@zainab-amir yes, it should be. Let me do that.

Reply:
done